saleor/graphql/schema.graphql

```diff
@@ -2151,7 +2398,7 @@ type Query {
   checkouts(before: String, after: String, first: Int, last: Int): CheckoutCountableConnection
   checkoutLine(id: ID): CheckoutLine
   checkoutLines(before: String, after: String, first: Int, last: Int): CheckoutLineCountableConnection
-  addressValidator(input: AddressValidationInput!): AddressValidationData
```

Comment:
Should it be deprecated instead of removed?

Reply:
I decided to just change the query in the schema since it was not used either in Dashboard 2.0 or the PWA storefront. In the long run, we'll be more strict about using deprecations when our API reaches a stable level. Right now we consider it still the beta version.